apollo-training/src/datasource/Trainee.js

```diff
@@ -0,0 +1,26 @@
+import { RESTDataSource } from 'apollo-datasource-rest';
+
+import config from '../config/configurations';
+
+export class TraineeAPI extends RESTDataSource {
+  constructor() {
```

Comment:
What is RESTDataSource

Reply:
We can get the apollo server context via this.context in the datasource class and get dataSources via this.context.dataSources. So we are extending RESTDataSource.